install/kubernetes/helm/istio-remote/templates/sidecar-injector-configmap.yaml

```diff
@@ -157,7 +157,16 @@ data:
         - mountPath: /etc/certs/
           name: istio-certs
           readOnly: true
+      {{ if .Values.global.sdsAPIEnabled }}
```

Comment:
Do we need both /etc/certs and /var/run/sds ? Why ?

Reply:
control plane components still have dependency on cidetal, will try to get rid of ```/etc/certs``` later.